src/renderer/main.tsx

```diff
@@ -61,9 +62,37 @@ export const Main = memo(() => {
         )
     );
 
+    const [status, setStatus] = useState(t('splash.loading', 'Loading...'));
```

Comment:
I think we should replace all `splash.x` keys with `setup.x` (or similar). What do you think?

Reply:
Sure